src/spaces/spaces.njk

```diff
@@ -57,50 +57,51 @@
     </div>
   </details>
 
+  <div class="govuk-o-grid">
   {% for space in spaces %}
-    <div class="govuk-o-grid space">
-      <div class="govuk-o-grid__item govuk-o-grid__item--one-third">
-        <h3 class="govuk-heading-m">
-          <span class="govuk-caption-m">
-            Space:
-          </span>
-
-          <a href="{{ linkTo('admin.organizations.spaces.view', {organizationGUID: organization.metadata.guid, spaceGUID: space.metadata.guid}) }}" id="space-{{ space.metadata.guid }}" class="govuk-link govuk-!-w-bold">
+    <a href="{{ linkTo('admin.organizations.spaces.view', {organizationGUID: organization.metadata.guid, spaceGUID: space.metadata.guid}) }}" id="space-{{ space.metadata.guid }}">
```

Comment:
Can we not wrap the entire section in a link? I realise it's how it's done in the design, but I have some flashing lights in my head which I cannot really explain ;/
It has only became OK with modern browsers and HTML5, which I think may mean problems with some readers, parsers or renderers.

Good example is Firefox, it selects additional parts of the page on `:focus`. I'm sure it could be solved with `a { display: block; }` or something, but I'm wary of possible edge cases.

I'd rather it be done with the use of JS if needed. `if box clicked, find link and execute`. But that's out of scope.

I'm not going to fight for it though... What do you think? /cc @carolinegreen @chrisfarms

Reply:
This is kind of the problem I was mentioning.  It's not the most accessible pattern either.  But the problem is that if it's implemented the way it was before, it's not necessarily obvious that the user can go to the next section with one small link.  We could really do with testing whether people would know to go to the next page.

Reply:
You're right. But aren't we trying to fix the problem that doesn't exist yet? We didn't hear problem about this yet from the people who have tested it.

Whilst you know it's a problem for readers and while I use firefox, it does have weird behaviour 🤔

Reply:
I've seen this happen on other interfaces with small links like this.  It doesn't make sense that there's a tiny link and that it goes somewhere, with the way it was implemented and we never discussed it with design and content.